.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+env:
+  RUST_BACKTRACE: 1
+  CARGO_TERM_COLOR: always
+
+jobs:
+  test-action:
+    name: Test the action
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout test repository
+        uses: actions/checkout@v3
+        with:
+          repository: mgr0dzicki/cargo-semver-action-ref-slice
+          fetch-depth: 0
+          persist-credentials: true
+      - name: Checkout action
+        uses: actions/checkout@v3
+        with:
+          path: action
+      # test without semver violation
+      - name: Checkout branch patch_change
+        run: git checkout patch_change
+      - name: Run action
```

Comment:
This name is what shows up in the GitHub Actions log, so it's an opportunity to put additional useful information rather than repeating the same string twice. The nearby comment has some good information that would be nice to see here! Try to work it in.